NodeBase/entry_point.sh

```diff
@@ -10,8 +10,20 @@ if [ ! -e /opt/selenium/config.json ]; then
   exit 1
 fi
 
+# In the long term the idea is to remove $HUB_PORT_4444_TCP_ADDR and $HUB_PORT_4444_TCP_PORT and only work with
+# $HUB_HOST and $HUB_PORT
+if [ ! -z "$HUB_HOST" ]; then
+  HUB_PORT_PARAM=4444
```

Comment:
confused about a couple of things, why do you check 

"if $HUB_HOST isn't empty, set port to 4444"

also, i see 4 variables...

HUB_HOST, HUB_PORT, HUB_HOST_PARAM, HUB_PORT_PARAM.

what's the differences between these? can';t we consolidate? make things a little simpler?

Reply:
No, no... There are only 3 variables: `HUB_HOST`, `HUB_PORT` and `HUB_PORT_PARAM`.


What I tried to do is to check first if `HUB_HOST` was set, if so then use `HUB_PORT_PARAM` as a placeholder for port 4444 (assuming that the user started the hub with port 4444). 

Therefore they only will need to set `HUB_HOST` and the default port will be 4444. And in case they started the hub with a different port, then `HUB_PORT` would be set and `HUB_PORT_PARAM` takes its value.

It could be simplified by doing something like:

```sh
if [ ! -z "$HUB_HOST" ] && [ ! -z "$HUB_PORT" ]; then
  echo "Connecting to the Hub using the host ${HUB_HOST} and port ${HUB_PORT}"
  HUB_PORT_4444_TCP_ADDR=${HUB_HOST}
  HUB_PORT_4444_TCP_PORT=${HUB_PORT}
fi
```

So the user will need to set both variables always. I just thought it could be nicer when the user only needs to set `HUB_HOST` since in most of the cases people will use 444 4 for hub.

What do you think?